File: vxpt-user/vxpt-user-api/src/main/java/com/github/blanexie/vxpt/api/user/feign/UserRpcFeign.java
package com.github.blanexie.vxpt.api.user.feign;

import com.github.blanexie.vxpt.api.user.UserRpc;
import org.springframework.cloud.openfeign.FeignClient;

@FeignClient(name = "vxpt-user", contextId = "UserRpcFeign")
public interface UserRpcFeign extends UserRpc {
}

File: vxpt-bbs/vxpt-bbs-server/src/main/java/com/github/blanexie/vxpt/bbs/feign/UserRpcFeign.kt
package com.github.blanexie.vxpt.bbs.feign

import com.github.blanexie.vxpt.api.user.feign.UserRpc
import org.springframework.cloud.openfeign.FeignClient

@FeignClient("VXPT-USER")
interface UserRpcFeign : UserRpc {
}